README.md

```diff
@@ -101,12 +105,14 @@ troi --help
 
 ### Installation for Development
 
+Note: If you have trouble installing nmslib, it is optional. See above.
```

Comment:
Probably add the command to install troi without nmslib here `pip3 install -e .[tests]`

Reply:
ok, will fix, there are a couple of problems here.